cpp/modmesh/view/RManager.cpp

```diff
@@ -184,6 +184,8 @@ void RManager::setUpMenu()
         this->addApplication(QString("euler1d"));
         this->addApplication(QString("linear_wave"));
         this->addApplication(QString("bad_euler1d"));
+        this->addApplication(QString("cavity2d_noQt"));
```

Comment:
`cavity2d` and `cavity2d_noQt` look similar and should be merged into one module/app.